pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "scip-clang"
+version = "0.0.0"
+description = "A SCIP indexer for C and C++ code."
+authors = ["Varun Gandhi <varun.gandhi@sourcegraph.com>"]
+license = "Apache-2.0"
+
+[tool.poetry.dependencies]
+python = "^3.9"
```

Comment:
You seem to pin something to 3.10 above, should those be the same?

Reply:
You mean in the CI code? I added that as a temporary attempt, since it seemed like the agent should have Python 3.10 (but it didn't work). Normally, we should not need that. This might also go away since @jhchabran was considering replacing this with `rules_python`.

Reply:
Resolving since this file is gone now.